### ISSUE TITLE
Can I pretrain the 13B model on 40GB A100 GPU?

### ISSUE THREAD

**Issue by mmaaz60** [haotian-liu/LLaVA#155]
### When did you clone our code?

I cloned the code base after 5/1/23

### Describe the issue

Issue:
I am trying to pretrain the LLaVA-13B-v1.1 model on 8 A100 40 GB GPUs.
Command:
```
PASTE THE COMMANDS HERE.
```
Same as mentioned in the README.
Log: 
```
PASTE THE LOGS HERE.
```
It gave OOM error while attempting to transfer the model on GPU. I am not using FSDP.
Screenshots:
You may attach screenshots if it better explains the issue.

**Comment by haotian-liu** [comment]
Hi @mmaaz60 can you please paste your command and the logs here?  It will be helpful for understanding the configuration (e.g. batch size, gradient accumulation, dataset, etc.) that can all contribute to the OOM issue.  Thanks!

**Comment by mmaaz60** [comment]
FSDP is required for 40GB GPU
